### PR TITLE
updated kochava to 3.4.0 to fix anrs

### PR DESCRIPTION
**Pull request by vivekb91**

### CODE REVIEW

File: android/config.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "libraries": [
   ],
   "jars": [
-    "libs/KochavaSDK.jar"
+    "libs/tracker-3.4.0.jar",
+    "libs/install-referer-1.0.jar"
```
Comment (rampr) [inline android/config.json:9]: is this needed ?

Comment (vivekb91) [reply android/config.json:9]: yes. needed https://support.kochava.com/sdk-integration/sdk-kochavatracker-android/migration-guide

File: android/manifest.xml [hunk 2]
```diff
@@ -7,9 +7,10 @@
 <!-- GameThrive -->
```
Comment (rampr) [inline android/manifest.xml]: please remove this or modify accordingly. This is a copy paste issue

File: android/manifest.xml [hunk 3]
```diff
@@ -7,9 +7,10 @@
...
 <meta-data android:name="kochavaAppGUID" android:value="" />
 
-<receiver android:name="com.kochava.android.tracker.ReferralCapture" android:exported="true" >
+<receiver android:name ="com.kochava.base.ReferralReceiver"
+        android:exported ="true">
   <intent-filter>
-    <action android:name="com.android.vending.INSTALL_REFERRER" />
+    <action android:name ="com.android.vending.INSTALL_REFERRER" />
```
Comment (rampr) [inline android/manifest.xml]: extra space before = and in the previous lines ?

**Pull request merged**